app/common/middleware/request.js

```diff
@@ -0,0 +1,14 @@
+const cookies = require('cookie-parser')
+const express = require('express')
+
+/**
+ * Request handling
+ */
+const request = [
```

Comment:
Not 100% sure about this file as it's just bringing in middleware from dependencies?

Could we just include these in `middleware/index.js` instead?

Reply:
Yeah that's an option too

I'll explain my thinking in terms of testing GET or POST requests:

**HTTP GET request**

1. Cookies are added to `req.cookies`
2. Form submission empty `req.body`

**HTTP POST request**

1. Cookies are added to `req.cookies`
2. Form submission empty `req.body`

We're left with a nice way to test everything "HTTP request related" in one middleware

Hypothetically you'd put things like **Content-Encoding** `gzip` and `application/json` handling here too

```mjs
import compression from 'compression'

export const request = [
  compression(),
  express.json(),
  express.urlencoded({
    extended: true
  })
]
```

Reply:
I've added tests for this